pwiz_tools/Skyline/Model/Lib/NistLibSpec.cs

```diff
@@ -871,6 +874,14 @@ private bool CreateCache(ILoadMonitor loader, IProgressStatus status, int percen
                             continue;
                         }
 
+                        match = REGEX_RI_LINE.Match(line);
+                        if (match.Success)
+                        {
+                            rt = GetRetentionTime(match.Groups[1].Value, true); // Note using RT as stand-in for RI (retention Time vs retention Index, LC vs GC)

```

Comment:
There is precedent for this. We store iRT values as RT values with Prosit and Assay Libraries. This only gets a bit weird with the Show IDs feature "From Other Runs" which plots the RT values from the library regardless of run matching. Then we end up plotting iRTs on an RT axis. I suppose we could consider some kind of flag in the library indicating it is an iRT and never to plot it on a Measured RT axis. But, that can be "FUTURE" work. This is not the first case.

Reply:
OK, I've added a CONSIDER comment that suggests we might someday want to track RI and RT simultaneously.